app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is a class variable, which means it is shared across all instances of the `Person` class. This can lead to unexpected behavior if multiple instances are created and modified. Consider using instance variables or a different approach to manage the relationships.